src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -4407,14 +4407,13 @@ IsPartitionColumn(Expr *columnExpression, Query *query)
  */
 void
 FindReferencedTableColumn(Expr *columnExpression, List *parentQueryList, Query *query,
-						  Oid *relationId, Var **column)
+						  Var **column, RangeTblEntry **rte)
```

Comment:
`FindReferencedTableColumn` / `IsPartitionColumn` seems to be used in several places. I think we should add some regression tests touching those code-paths.

Reply:
minor suggestion: `rte` -> `rteContainingReferencedColumn`